src/Text/Pandoc/Readers/JATS.hs

```diff
@@ -15,7 +15,7 @@ Conversion of JATS XML to 'Pandoc' document.
 
 module Text.Pandoc.Readers.JATS ( readJATS ) where
 import Control.Monad.State.Strict ( StateT(runStateT), gets, modify )
-import Control.Monad (forM_,  when, unless, MonadPlus(mplus))
```

Comment:
`mplus` is now redundant, since line dealing with the non-existing`info` element is redundant and was removed.